textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/queryrewriter/WordDictionary.java

```diff
@@ -0,0 +1,31 @@
+package edu.uci.ics.textdb.dataflow.queryrewriter;
+
+import edu.uci.ics.textdb.api.common.ITuple;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashSet;
+
+/**
+ * Created by shiladitya on 4/25/16.
+ * Class to provide simple abstraction of a dictionary of words with constant retrieval time
+ * Used by FuzzyTokenizer to process query
+ */
+public class WordDictionary {
```

Comment:
For now, since it is an implementation class it is being removed. We will take the discussion up with Team 1, afk.